klippy/toolhead.py

```diff
@@ -569,12 +604,13 @@ def check_busy(self, eventtime):
     def get_status(self, eventtime):
         print_time = self.print_time
         estimated_print_time = self.mcu.estimated_print_time(eventtime)
+        extruder = self.extra_axes[0]
         res = dict(self.kin.get_status(eventtime))
         res.update({ 'print_time': print_time,
                      'stalls': self.print_stall,
                      'estimated_print_time': estimated_print_time,
-                     'extruder': self.extruder.get_name(),
-                     'position': self.Coord(*self.commanded_pos),
+                     'extruder': extruder.get_name(),
+                     'position': self.Coord(*self.commanded_pos[:4]),
```

Comment:
Would it be helpful to report the positions of other extra axes, e.g. as `extra_axis.<axis_name>`?

Reply:
I agree that sounds useful.  My rough plan was to add the support and then look to see what values to export.